### PR TITLE
Fixed issue #16958: User with roles can not have more rights

### PR DESCRIPTION
**Pull request by gabrieljenik**
Added a warning message to the assignment of permissions. Modified the controller so that it effectively cleans up the roles when you assign individual permissions.

### CODE REVIEW

File: application/controllers/UserManagementController.php [hunk 1]
```diff
@@ -416,6 +416,9 @@ public function actionSaveUserPermissions()
         }
         $userId = Yii::app()->request->getPost('userid');
         $aPermissions = Yii::app()->request->getPost('Permission', []);
+
+        $clearUser = Permissiontemplates::model()->clearUser($userId);
```
Comment (olleharstedt) [inline application/controllers/UserManagementController.php:420]: Is this variable used anywhere?

Comment (gabrieljenik) [reply application/controllers/UserManagementController.php:420]: Removed

Comment (olleharstedt) [reply application/controllers/UserManagementController.php:420]: Is it removed? Or am I stupid?

**Comment by Shnoulle** [comment]
Why not disable all checkbox ?

More clear  (_user didn't read_)

File: application/controllers/UserManagementController.php (hunk 1 above)
Comment (gabrieljenik) [reply application/controllers/UserManagementController.php]: I am. Code commited wasn't pushed

**Comment by olleharstedt** [comment]
Thanks!

**Pull request merged**